x-pack/plugins/audit_trail/kibana.json

```diff
@@ -0,0 +1,10 @@
+{
+  "id": "auditTrail",
+  "version": "8.0.0",
+  "kibanaVersion": "kibana",
+  "configPath": ["xpack", "audit_trail"],
+  "server": true,
+  "ui": false,
+  "requiredPlugins": ["licensing", "security", "spaces"],
```

Comment:
Should `security` and `spaces` be optional? I mean, shouldn't the auditTrail factory be capable of emitting 'degraged' events with only the info from the enabled sources?

Reply:
@pgayvallet probably. I left detailed implementation to the Security team, but we can improve this in the current PR